src/App.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import Router from './routes/Router';
 import { AuthProvider } from './context/AuthProvider';
 import MainLayout from './layouts/MainLayout';
+import Footer from "./layouts/Footer";
```

Comment:
If you look at the file `src\layouts\MainLayout.jsx` or (`src/layouts/MainLayout.jsx` if you are on Mac) you'll see that the footer is already imported and used there so there is no need for you to import it in `App.jsx`. Also, I'm pretty sure that we have a linter that doesn't allow it that you only import a component or create a variable and never use it. So, you can remove Footer from `App.jsx`.

Reply:
Thanks. Updated and reolved